IBM/carbon-addons-iot-react
# [Table] RTL  and Table filters combo box bug fixes

Closes #446 closes #1108

**Summary**

- several RTL issues with various pieces of the data Table outlined in the comments below
- bug fix for combo box that are used in our data table filter

**Acceptance Test (how to verify the PR)**

- Check areas of the table outlined in the comments below. Most defects can be found with [this](https://deploy-preview-1112--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-table--with-no-data) story
- open up knob panel and activate RTL mode
- You should see all areas behave properly in RTL mode
